### PR TITLE
Milestone 2: Portfolio finish mobile version

### PR DESCRIPTION
**Pull request by Chandan-devs-tech**
## In this Milestone my Learning objectives :

- Perfect my HTML semantics best practices knowledge
- Perfect my CSS3 layouts using Flexbox to place elements on the page.
- Perfect my knowledge of GitHub flow
- Perfect my professional documentation knowledge
- Was able to understand how to plan my HTML to come up with the desired markup
- Understand how to parse a Figma design to create a UI.
- Use CSS Grid to place elements on the page.
- Build a personal portfolio mobile site.
- Added [IMAGE- Portfolio-Finish-Mobile-Version(iPhone SE).png]

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,40 +1,236 @@
...
+      <div class="about-container">
+        <div class="about-content">
+          <h2 class="about-title">About Myself</h2>
+          <p class="about-para">
+            Hello I'm a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don't
+            hestiate to contact me.
+          </p>
+          <a class="about-contact" href="#">LET'S CONNECT</a>
```
Comment (Kossi-stack) [inline index.html]: - [ ] Kindly use header tag hx

File: index.html [hunk 2]
```diff
@@ -1,40 +1,236 @@
...
+                <img src="Assets/Ellipse CSS3.svg" alt="CSS3-Logo" />
+                <h4 class="langs-text">CSS</h4>
+              </li>
+            </ul>
+          </div>
+          <div>
+            <div class="framework">
+              <h4 class="frame-title">Frameworks</h4>
+
+              <img src="Assets/Union Left.svg" alt="arrow" />
```
Comment (Kossi-stack) [inline index.html]: - [ ] Kindly wrap the technology categories in ul each of them in li

File: index.html [hunk 3]
```diff
@@ -1,40 +1,236 @@
...
+              <img src="Assets/Union Down.svg" alt="arrow" />
+            </div>
+
+            <ul class="lang">
+              <li class="langs">
+                <img
+                  src="Assets/Ellipse JavaScript.svg"
+                  alt="JavaScript-Logo"
+                />
+                <h4 class="langs-text">JavaSript</h4>
```
Comment (Kossi-stack) [inline index.html:205]: - [ ] Kindly wrap the technology in each category in ul and each of them in a li

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @Chandan-devs-tech,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Required Changes ♻️

 - [ ] Kindly fix the margin according to figma design
 Figma design

Screenshot 2023-01-14 at 13 36 21

Your Design

Screenshot 2023-01-14 at 13 38 28

 - [ ] Kindly make your social handle in the readme a link.

 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by algerina — APPROVED** [review]
Hi @Chandan-devs-tech  ,
Good job so far! 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- Your project looks awesome:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
You did great implementing the previews TSE's requirements :+1: 
Your project is complete!  It's time to merge it :shipit:



 
 ## Optional suggestions
 
 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

**Pull request merged**